recipes/taskgraph/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
 
 build:
     number: 0
-    script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
You want to leave the `script` line that you deleted. Sorry for the confusion is I was unclear in my original comment.

Reply:
Whoops!  Thanks for clarifying that ... this is my first `noarch: python` package.  Patched in b75f78197